tracing/src/main/java/org/axonframework/extensions/tracing/TracingCommandGateway.java

```diff
@@ -90,7 +90,7 @@ protected TracingCommandGateway(Builder builder) {
     @Override
     public <C, R> void send(C command, CommandCallback<? super C, ? super R> callback) {
         CommandMessage<?> cmd = GenericCommandMessage.asCommandMessage(command);
-        sendWithSpan(tracer, "sendCommandMessage", cmd, (tracer, parentSpan, childSpan) -> {
+        sendWithSpan(tracer, "send"+ SpanUtils.messageName(cmd), cmd, (tracer, parentSpan, childSpan) -> {
```

Comment:
As mentioned in a comment on the `OpenTraceHandlerInterceptor`, I think we should add an underscore between `send` and the resolved message name.